.github/workflows/ci.yml

```
@@ -151,7 +149,18 @@ jobs:
           key: shards-${{ hashFiles('shard.lock') }}
 
       - name: Install Shards
-        run: shards install
+        run: |
+          if ! shards check; then
+            shards install
+          fi
+
+      - name: Run Crystal linter
```

```suggestion
      - name: Check Crystal formatter compliance
```

Since we have an actual linter now with Ameba I think we should probably be specific that the Crystal linter is just the formatter